src/evaluation/test_validation.py

```diff
@@ -0,0 +1,50 @@
+import pytest
```

Comment:
The directory structure is a bit confusing. There is a `test_data` folder under `data` and under `src/evaluation`. It seems for this test only the latter is used. What is the first one for?
In general, it is good practice to keep all tests in a separate  `test` folder (e.g. at the same level as the `src` folder). The test data can then live in a `data` folder in the `test` folder.

Reply:
The many different test data files makes it difficult to see which data is actually used for this test. You could zip the separate files and name it `test_validation_data.zip` so that it is clear which data is used for this specific test.

Reply:
Thanks for the tips @hailihu  I created a separate test folder. I added the corresponding data set (renamed as "test_validation_data")  to this folder